samples/bot-request-approval/csharp/README.md

```diff
@@ -0,0 +1,114 @@
+---
+page_type: sample
+description: This sample shows a feature where user can send task request to his manager and manager can approve/reject the request in group chat.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- csharp
+extensions:
+contentType: samples
+createdDate: "13-12-2021 17:00:25"
+---
+
+# Send task request using universal adaptive cards in group chat
+
+This sample shows a feature where:
+1. Requester : Can request for any task approval from manager by intiating request in group chat using bot command `request` and only requester can edit the request card.
+2. Manager : Can see the request raised by user in the same group chat with an option of approve or reject.
+3. Others: Other members in the group chat can see the request details only.
+
+Requester:
+
+- Initiated request using bot command `request` in group chat.
+
+  ![Initial Card](BotRequestApproval/Images/InitialCard.png)
+
+- Card will refresh for requester to fill details.
+
+  ![Request Card](BotRequestApproval/Images/RequestCard.png)
+  
+- After submitting the request, requester can edit or cancel the request.
+
+  ![Edit/Cancel Card](BotRequestApproval/Images/EditCancelCard.png)
+
+Manager:
+
+- After requester submit the request, manager can approve/reject the request.
+  Note: Only the manager of the request will see the buttons to approve/reject the request.
```

Comment:
This should come in next line:
![image](https://user-images.githubusercontent.com/59052547/145363727-d197b78a-cadd-465c-ab09-7eabdf21d71e.png)